Content.Shared/_DV/CosmicCult/Components/Examine/CosmicMarkBlankComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.StatusIcon;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Shared._DV.CosmicCult.Components.Examine;
+
+/// <summary>
+/// Marker component for targets under the effect of Shunt Subjectivity or Astral Projection.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class CosmicMarkBlankComponent : Component
```

Comment:
If the `Mark` in here means `Marker`, it's superfluous, and I think incorrect since it has data
```suggestion
public sealed partial class CosmicMarkBlankComponent : Component
```